test/bin/get_latest_rpm_version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# This script reports the version of the latest RPM in the local
+# repository.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+release_info_rpm=$(find "${LOCAL_REPO}" -name 'microshift-release-info-*.rpm')
+if [ -z "${release_info_rpm}" ]; then
+    error "Failed to find microshift-release-info RPM in ${LOCAL_REPO}"
+    exit 1
+fi
+rpm -q --queryformat '%{version}' "${release_info_rpm}"
```

Comment:
If we assume x.y.z version format, should we verify it here to avoid surprises? 
Note also that this command may produce warnings.

```
$ rpm -q  --queryformat '%{version}' microshift-release-info-4.14.0~ec.3-202307062134.p0.g0ba2d5c.assembly.ec.3.el9.noarch.rpm
warning: microshift-release-info-4.14.0~ec.3-202307062134.p0.g0ba2d5c.assembly.ec.3.el9.noarch.rpm: Header V3 RSA/SHA256 Signature, key ID 5cd7bf6a: NOKEY
4.14.0~ec.3
```

Reply:
Those warnings go to stderr, right?

The full string is the version. We need the actual full value in some cases and only the minor version in others. It's up to the caller to parse it.

Reply:
I would at least add `2>/dev/null` redirection to filter out unnecessary information.

Reply:
OK